src/main/java/me/botsko/prism/players/PlayerIdentification.java

```diff
@@ -301,7 +300,6 @@ protected static void updatePlayer( PrismPlayer prismPlayer ){
         } catch (SQLException e) {
             e.printStackTrace();
         } finally {
-            if(rs != null) try { rs.close(); } catch (SQLException e) {}
```

Comment:
it was removed because it is never used....take a look we never actually get a result set...just execute the prepared statement....